.github/docker/Dockerfile.runner-packages

```
@@ -0,0 +1,14 @@
+FROM ubuntu:22.04
```

`24.04` is the current LTS Ubuntu. Are you able to use that or does it have to be `22.04`?

```suggestion
FROM ubuntu:24.04
```